app/app.test.js

```diff
@@ -51,10 +51,13 @@ describe(`http://localhost:${PORT}`, () => {
     it('should display the list of components', async () => {
       const response = await fetchPath('/')
       const $ = cheerio.load(await response.text())
+
+      const componentsDirectory = await getDirectories(configPaths.components)
       const componentsList = $('li a[href^="/components/"]').get()
+
       // Since we have an 'all' component link that renders the default example of all
       // components, there will always be one more expected link.
-      const expectedComponentLinks = lib.allComponents.length + 1
+      const expectedComponentLinks = componentsDirectory.size + 1
```

Comment:
The file system helpers `getDirectories()`, `getFiles()` or the "everything" `getListing()` are now [Maps](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map)

```mjs
Map {
  'component-1' => {
    basename: 'component-1',
    path: 'src/govuk/components/component-1',
    stats: fs.Stats
  },
  'component-2' => {
    basename: 'component-2',
    path: 'src/govuk/components/component-2',
    stats: fs.Stats
  }
}
```

Which comes in handy for simplifying lots of other tests:

**`componentsDirectory.keys()`** → `['component-1', 'component-2']`
**`componentsDirectory.has('component-2')`** → `true`
**`componentsDirectory.has('component-3')`** → `false`